openapi/Swarm.yaml

```
@@ -867,6 +867,30 @@ paths:
           description: Default response
 
   "/stewardship/{reference}":
+    get:
+      summary: "Check if content is available"
+      tags:
+        - Stewardship
+      parameters:
+        - in: path
+          name: reference
+          schema:
+            $ref: "SwarmCommon.yaml#/components/schemas/SwarmReference"
+          required: true
+          description: "Root hash of content (can be of any type: collection, file, chunk)"
+      responses:
+        "200":
+          description: Returns an individual postage batch state
```

That does not seem correct. 
```suggestion
          description: Returns if the content is retrievable
```
```suggestion
          description: Returns an individual postage batch state
```